app/views/generic_object_definition/_show_custom_button.html.haml

```diff
@@ -27,10 +27,5 @@
       .col-md-8
         %span{:class => @record.options[:button_icon], :style => @record.options.key?(:button_color) ? "color: #{@record.options[:button_color]};" : nil}
 
-  %generic-object-definition-toolbar#generic_object_definition_show_form{'record-id'    => @record.id,
```

Comment:
Is there a angular controller file to this or is it already removed?

Reply:
I am still seeing app/javascript/oldjs/components/generic_object/generic-object-definition-toolbar.js